Add files via upload

brownfrosamurai opened the pull request:
uxDesign assignment was added by meduna oluwafemi

brownfrosamurai committed:
Add files via upload

git and version control assignment was added by meduna oluwafrmi

otaruMendez commented:
Great job! @brownfrosamurai

goke-epapa reviewed (commented):
Well done @brownfrosamurai.

A few comments:

1. Please update the title of your Pull Request (PR) to reflect the changes you made.
For example `UX Design Task` or `Git and Version Control Task`
2. Your folder structure doesn't seem to fit the requirements, I see you have a `Git and version control` folder, then you have a folder named `meduna oluwafemi`. I believe the folder with your name should be directly in the `submissions` folder. can you please clarify?

brownfrosamurai committed:
Add files via upload

Files added by meduna oluwafemi include. 
1. HTML assignment.
2. Requirement gathering assignment.
3. Wireframes for my e-commerce website.

brownfrosamurai commented:
Hi Goke,

I have have made the necessary changes to my pull request and I have also
made a new pull request with my most recent assignment.

I had to upload the files using the 'drag and drop' method, I hope this is
just as acceptable.

I ran into some problems as regards using git bash to commit and add files
to my github account, Can you help me in sorting out this problem?

Regards.

On Thu, Oct 24, 2019 at 3:25 PM Goke Obasa <notifications@github.com> wrote:

> *@goke-epapa* commented on this pull request.
>
> Well done @brownfrosamurai <https://github.com/brownfrosamurai>.
>
> A few comments:
>
>    1. Please update the title of your Pull Request (PR) to reflect the
>    changes you made.
>    For example UX Design Task or Git and Version Control Task
>    2. Your folder structure doesn't seem to fit the requirements, I see
>    you have a Git and version control folder, then you have a folder
>    named meduna oluwafemi. I believe the folder with your name should be
>    directly in the submissions folder. can you please clarify?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/pull/18?email_source=notifications&email_token=ANHIBV54XWPJSUMZ6DXP6P3QQIOGDA5CNFSM4JCWC75KYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCJFOKMY#pullrequestreview-306898227>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANHIBV7OSSU2MSOQHNCHRPLQQIOGDANCNFSM4JCWC75A>
> .
>

brownfrosamurai committed:
HTML for a sign up page was added by meduna femi.

adaugochi commented:
Hello, you are submitting two index.html files, why? Also, your file directory should be `submissions folder->your name folder-> name of the task (e.g HTML, UxDesign etc) folder-> the file you want to submit`.

brownfrosamurai commented:
Hi Maryfaith,

The first assignment had some mistakes which i corrected in the second HTML
assignment I submitted.

I have tried a few things in attempts to fix the order of the directory,
but I have not been able to do so, Can you help me understand what to do.

On Mon, Oct 28, 2019 at 8:28 AM Maryfaith Mgbede <notifications@github.com>
wrote:

> Hello, you are submitting two index.html files, why? Also, your file
> directory should be submissions folder->your name folder-> name of the
> task (e.g HTML, UxDesign etc) folder-> the file you want to submit.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/pull/18?email_source=notifications&email_token=ANHIBV5BV5OV3ZVDYLJJFO3QQ4ALDA5CNFSM4JCWC75KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOECNJIIY#issuecomment-547001379>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANHIBV2XMHBYK3QL2P7XRITQQ4ALDANCNFSM4JCWC75A>
> .
>

adaugochi commented:
@brownfrosamurai can you send me a DM on slack (Adaa)

brownfrosamurai committed:
Add files via upload

html was updated and styled by meduna femi

tegaphilip commented:
@brownfrosamurai you should have just one html file. delete the others and push to the repo again so that the single html file can be properly reviewed.

brownfrosamurai committed:
Delete index.html

I deleted this html file because I have two of them

brownfrosamurai committed:
Delete index.html

I deleted this html file because I had two of them.

brownfrosamurai committed:
Add files via upload

An html folder which includes a styles folder and the image I used for the background was added by me.

yemexx1 commented on meduna oluwafemi/uxDesign/HTML/css/style.css:
```diff
@@ -0,0 +1,126 @@
+    color:white;
+}
+
+/* ::placeholder {
```
Please remove this comment

brownfrosamurai replied:
I have made the necessary changes.

yemexx1 commented on submissions/index.html:
```diff
@@ -0,0 +1,151 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>sign-up page</title>
+    <style>
```
Could you please extract this into a css file?

brownfrosamurai committed:
I deleted this html because I have duplicates.

brownfrosamurai committed:
I added a revised styling file.

tegaphilip commented:
@brownfrosamurai well done. However, now you have two css files. please delete one of them just like you did when you had multiple html files. Make sure the one you deleted is not the one your html code is pointing to. I think your html file is pointing to `meduna oluwafemi/uxDesign/HTML/css/style.css` so you should delete the other one which is `submissions/css/style.css`. 
Or is there a reason why you have two css files? Then push your code again.

Also update the title of your PR as suggested here. https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/pull/18#pullrequestreview-306898227

brownfrosamurai committed:
I deleted a duplicate of my css file.

JohnItoo reviewed (commented):
@brownfrosamurai  This is a reminder to attend to the comment on your `style.css` file. Please kindly remove the commented out block. Thanks.

brownfrosamurai committed:
Delete style.css

I deleted the comment referenced by the mentors.

brownfrosamurai committed:
Add files via upload

I  deleted the comment referenced by my mentors and added an updated style.css file
